From 151a8176c6648c85d2cfbc4d0618dd567ded60e4 Mon Sep 17 00:00:00 2001
From: Gareth Kitchen <gkitchen@cookability.biz>
Date: Fri, 22 Jun 2018 09:13:52 +0100
Subject: [PATCH 2/3] Add tip to compile Heimdall from source.

Current versions of the binaries at the official site
and distributed by apt-get are version 1.4.1 which
does not support models S5 or greater.

Change-Id: I9e56c3fd1ebb2787d1e4b3d34fbbba7a172988ec
---
 _includes/templates/recovery_install_heimdall.md | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/_includes/templates/recovery_install_heimdall.md b/_includes/templates/recovery_install_heimdall.md
index 804c6f3..d373b14 100644
--- a/_includes/templates/recovery_install_heimdall.md
+++ b/_includes/templates/recovery_install_heimdall.md
@@ -10,6 +10,8 @@ Samsung devices come with a unique boot mode called "Download mode", which is ve
 [Heimdall](http://www.glassechidna.com.au/products/heimdall/) is a cross-platform, open-source tool for interfacing with Download mode on Samsung devices.
 The preferred method of installing a custom recovery is through this boot mode{% unless custom_root_instructions %} -- rooting the stock firmware is neither necessary nor required{% endunless %}.
 
+{% include tip.html content="We would recommend that you compile the latest version of Heimdall from [source](https://gitlab.com/BenjaminDobell/Heimdall) to ensure more modern devices are supported." %}
+
 1. Download and install the [Heimdall suite](http://glassechidna.com.au/heimdall/#downloads):
     * **Windows**: Extract the Heimdall suite and take note of the directory containing `heimdall.exe`. You can verify Heimdall is working by opening a command
     prompt in that directory and typing `heimdall version`. If you receive an error, make sure you have the 
-- 
2.17.1

